pkg/skaffold/deploy/helm_test.go

```diff
@@ -565,6 +586,60 @@ func TestPackageHelmChart(t *testing.T) {
 
 }
 
+// TODO(tstromberg): Remove private function test once there are side-effects in a public function
+func TestBinVer(t *testing.T) {
+	tests := []struct {
```

Comment:
For completeness can we add a function where incorrect versions throw error as expected

Reply:
Done.